library/build.gradle

```diff
@@ -44,3 +45,19 @@ task androidSourcesJar(type: Jar) {
     classifier = 'sources'
     from android.sourceSets.main.java.srcDirs
 }
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                groupId = 'com.github.stemyke'
```

Comment:
Why would this publishing be included in this repo?

Reply:
@Dimezis I was just figuring how should I publish the package, so I can import it into the project I mentioned. This is actually my first time I am doing something with an Android lib.